packages/openapi2-parser/lib/adapter.js

```diff
@@ -29,9 +29,27 @@ function parse(options) {
   });
 }
 
+function validate(options) {
+  return new Promise((fulfil, reject) => {
+    const parser = new Parser(options);
+    parser.generateMessageBody = false;
+    parser.generateMessageBodySchema = false;
```

Comment:
Seems strange to put those props directly on the parser. Why not the following?
```suggestion
    const parser = new Parser({
      ...options,
      generateMessageBody: false,
      generateMessageBodySchema: false
    });
```